config/members-crossplane.yaml

```diff
@@ -439,3 +439,16 @@ spec:
     user: zzxwill
     organization: crossplane
     role: direct_member
+---
+apiVersion: organizations.github.crossplane.io/v1alpha1
+kind: Membership
+metadata:
+  name: crossplane-stevendborrelli
```

Comment:
@muvaf can we make this alphabetical in list as well?

Reply:
Done!